src/main/java/se/citerus/dddsample/interfaces/handling/ws/HandlingReport.java

```diff
@@ -0,0 +1,70 @@
+package se.citerus.dddsample.interfaces.handling.ws;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
```

Comment:
HandlingReport is used in both the reg-app and the core-app. Do you think this api-class should be moved to some api-module sometime in the future to improve clarity and avoid duplication?

Reply:
Yes, it would make sense to move it to a separate module, package that as a separate jar and import that into the regapp. I will add that to the Trello backlog.